tests/python/unit/test_ontology_tagger.py

```diff
@@ -74,12 +70,13 @@ def test_keyword_tokenization():
     )
     tags = ontology_df["tag"].values.tolist()
     keywords = ontology_df["keyword"].values.tolist()
-    patterns = tagger._get_patterns(keywords)
     tagger._initialize_tokenizer([tagger.language])
-    tagger._match_with_category(patterns, tags, keywords)
-    ruler = tagger.tokenizer.spacy_nlp_dict[tagger.language].get_pipe("entity_ruler")
-    for elt in ruler.patterns:
-        assert elt["pattern"] in tagger.keyword_to_tag["en"]
+    tagger._match_no_category(tags, keywords)
+    matcher = tagger.matcher_dict[tagger.language]
+    # ruler = tagger.tokenizer.spacy_nlp_dict[tagger.language].get_pipe("entity_ruler")
+    patterns = tagger._tokenize_keywords(tagger.language, tags, keywords)
```

Comment:
Why is this called patterns?

Reply:
The idea was to  rely on SpaCy terminology  https://spacy.io/usage/rule-based-matching#phrasematcher

Reply:
Okay